### DISCUSSION TITLE
Using no_proxy to access internal Artifactory/ GitLab Instance

### DISCUSSION THREAD

**Discussion by steffakasid** [goreleaser/goreleaser#2853] [Q&A] [answered]
Hi,

I'm trying to publish into a internal Artifacotry/ GitLab Instance. Does goreleaser respect any Proxy Settings set via HTTP_PROXY/HTTPS_PROXY/NO_PROXY environment variables? I didn't find anything related in the documentation.

Regards
Steffen

**Comment by caarlos0** [comment, marked as answer]
AFAIK the Go HTTP client honors these environment variables by default... so... it should work, but I personally never tested it.

**Reply by steffakasid** [reply]
Thanks a lot. Indeed it seems to work. I just couldn't get anything related in the logs when using debug and in the documentation I couldn't find something.

Regards
Steffen
